banzai/utils/fits_utils.py

```diff
@@ -45,6 +46,7 @@ def create_master_calibration_header(images):
     mean_dateobs = date_utils.mean_date(observation_dates)
 
     header['DATE-OBS'] = (date_utils.date_obs_to_string(mean_dateobs), '[UTC] Mean observation start time')
+    header['DATE-CRT'] = (date_utils.date_obs_to_string(datetime.datetime.utcnow()), '[UTC] Master creation time')
```

Comment:
Often, the convention is that DATE is the date the fits file was created.

Reply:
I switched to using `DATE`